PhysicsTools/NanoAOD/plugins/MuonVtxAgnosticIsoProducer.cc

```diff
@@ -0,0 +1,158 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/TrackExtraFwd.h"
+
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include <eigen3/Eigen/Dense>
```

Comment:
These includes aren't needed and cause issues for me, FYI

Reply:
Yah the Eigen one has the wrong paths given how the headers are arranged in CMSSW vs in the cmswmass singularity image.  Make a PR taking this out?